src/commissioner/Commissionee.h

```diff
@@ -0,0 +1,162 @@
+/*
+ *    Copyright (c) 2020-2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <controller/CHIPDeviceControllerSystemState.h>
+#include <controller/InvokeInteraction.h>
+#include <controller/ReadInteraction.h>
+
+namespace chip {
+namespace Commissioner {
+
+using namespace Controller;
+
+struct Commissionee
+{
+    DeviceControllerSystemState * mSystemState = nullptr;
+    chip::SessionHolder mPaseSession;
+    chip::SessionHolder mCaseSession;
+    chip::Optional<chip::Transport::PeerAddress> mCommissionableNodeAddress;
+    chip::Optional<chip::Transport::PeerAddress> mOperationalAddress;
+    chip::Optional<chip::ReliableMessageProtocolConfig> mMrpConfig;
+    chip::Optional<chip::PeerId> mOperationalId;
+#if CONFIG_NETWORK_LAYER_BLE
+    chip::Optional<BLE_CONNECTION_OBJECT> mBleConnection;
+#endif
```

Comment:
Style comment / optional:  When possible, I prefer to only use a macro like this once in a file.  You could rearrange the code here and have a simple no-op implementation for ClosePase() to only use the macro once.